CONTRIBUTORS

```diff
@@ -2,6 +2,7 @@ Heinrich Apfelmus <https://github.com/HeinrichApfelmus>
 Daniel Austin     <https://github.com/daniel-r-austin>
 Jeremy Barisch-Rooney <https://github.com/barischj>
 Steve Bigham
+brprice           <https://github.com/brprice>
```

Comment:
Would you possibly prefer to see a different name used here, @brprice?

Reply:
```suggestion
Ben Price         <https://github.com/brprice>
```